client/public/index.html

```diff
@@ -1,43 +1,104 @@
-<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
-    <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <meta name="theme-color" content="#000000" />
-    <meta
-      name="description"
-      content="Web site created using create-react-app"
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```

Comment:
Probably wise not to change the index.html: add the starter HTML to the App component instead.